pkg/enemy/types.go

```diff
@@ -0,0 +1,282 @@
+package enemy
+
+import (
+	"fmt"
+
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/info"
+	"github.com/genshinsim/gcsim/pkg/shortcut"
+)
+
+var levelMultiplier = [][]float64{
```

Comment:
There is a new hp level multiplier curve that was added in 4.0, see https://genshin-impact.fandom.com/wiki/Enemy/Level_Scaling#HP_Level_Multiplier, so the HP values listed for enemies like Underwater Survey Mek (underwatersurveymek) are wrong.